### DISCUSSION TITLE
incorrect output operation. error 405

### DISCUSSION THREAD

**Discussion by gigabait15** [community/community#110555] [New to GitHub]
hello everyone, there was a problem with the project, created the user application and registered the entry and exit buttons, but when you click on the exit button, error 405 crashes. another project with the same structure works correctly. I'm starting my way in development, so I don't know how to solve this problem. chatgpt can't help either.I am grateful in advance for your help.here is the link to the project: https://github.com/gigabait15/home_work_19_23.git. 
An online store project, the problem is with the exit button in the pop-up menu. here is the given html template 
```
<div class="col-sm-4 offset-md-1 py-4">
    <h4 class="text-white">Меню</h4>
    <ul class="list-unstyled">
        <li><a href="{% url 'catalog:home' %}" class="text-white">Главная</a></li>
        <li><a href="{% url 'catalog:contact' %}" class="text-white">Контакты</a></li>
        <li><a href="{% url 'blog:blog' %}" class="text-white">Блог</a></li>
        {% if user.is_authenticated %}
            <li><a href="{% url 'users:logout' %}" class="text-white">Выйти</a></li>  <-The problem with her
            <li><a href="{% url 'users:profile' %}" class="text-white">Профиль</a></li>
        {% else %}
            <li><a href="{% url 'users:login' %}" class="text-white">Войти</a></li>
            <li><a href="{% url 'users:register' %}" class="text-white">Регистрация</a></li>
        {% endif %}
    </ul>
</div>
```

**Comment by ettaboyle** [comment]
Hi @gigabait15, we're happy you're here! 

You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction., we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by majcek210** [comment]
Try using a form with POST instead of an `<a>` tag.  
I tried  my best and I hope it helps!

```html
<form method="post" action="{% url 'users:logout' %}">
    {% csrf_token %}
    <button type="submit" class="btn btn-link text-white">Выйти</button>
</form>
